cedar-policy-validator/src/schema/namespace_def.rs

```diff
@@ -139,6 +140,192 @@ pub struct ActionFragment {
     pub(super) attributes: HashMap<SmolStr, PartialValueSerializedAsExpr>,
 }
 
+/// A common type reference resolver
+#[derive(Debug)]
+pub(crate) struct CommonTypeResolver<'a> {
```

Comment:
This struct would make more sense in the main `schema.rs` file or in its own submodule IMO

Reply:
Good idea. Done.